src/ActionPanel.scss

```diff
@@ -1,57 +1,91 @@
 @import 'variables';
 
-// TODO: Rework the arrow implementation. For example, it is not quite
-//       centered with respect to the action panel.
-
-.ActionPanel__panel {
-    width: $c2lc-action-panel-width;
+.ActionPanel__background {
+    align-self: center;
     height: $c2lc-action-panel-height;
-    background-color: #F1F1F1;
-    border: 0.1rem solid #939595;
-    border-radius: 3rem;
-    display: flex;
-    justify-content: space-evenly;
-    align-items: center;
-}
-
-.ActionPanel__panel::before {
-    content: '';
     position: absolute;
-    left: 45%;
-    top: 100%;
-    border-left: 0.7rem solid transparent;
-    border-top: 1rem solid #F1F1F1;
-    border-right: 0.7rem solid transparent;
+    top: -0.3rem;
+    width: $c2lc-action-panel-width;
     z-index: 1;
+
+    svg {
+        width: $c2lc-action-panel-width;
+        path {
+            fill: #30444E;
+            stroke: #4C9990;
+        }
+    }
 }
 
-.ActionPanel__panel::after {
-    content: '';
+.ActionPanel__panel {
+    align-items: center;
+    background-color: transparent;
+    display: flex;
+    height: $c2lc-action-panel-height;
+    justify-content: space-evenly;
+    padding-left: 2rem;
+    padding-right: 2rem;
     position: absolute;
-    left: 43.8%;
-    top: 100%;
-    border-left: 0.9rem solid transparent;
-    border-top: 1.2rem solid #939595;
-    border-right: 0.9rem solid transparent;
+    width: $c2lc-action-panel-width;
+    z-index: 2;
 }
 
 .ActionPanel__panel--hidden {
     display: none;
 }
 
 button.btn.ActionPanel__action-buttons {
-    background-color: #8F8F8F;
-    width: 3rem;
-    height: 3rem;
-    border-radius: 50%;
-    padding: 0.3rem;
+    background-color: #4C9990;
+    border: none;
+    border-radius: 50%;    
+    height: 2.1rem;
```

Comment:
We have been ensuring that all interactable items are at least 45px by 45px to meet accessibility requirements, but I noticed that the new stop button is designed to be smaller than that as well. Could you please confirm with Sepideh about this? If she thinks we need to meet 45px requirement, height and width needs to be changed to 2.5rem.